projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## HEAD (unreleased)
 
+- Enhancement: Add validation when change on `minDate` or `maxDate` occurs.
```

Comment:
Can we add some detail regarding what component we are updating?

Reply:
Yes, we can. Added!